jupyter_fsspec/handlers.py

```diff
@@ -251,6 +251,77 @@ def post(self):
         self.finish()
 
 
+# ====================================================================================
+# Handle Syncing Local and Remote filesystems
+# ====================================================================================
+class FilesystemSyncHandler(BaseFileSystemHandler):
+    def initialize(self, fs_manager):
+        self.fs_manager = fs_manager
+
+    async def get(self):
+        # remote to local (fetching latest changes)
```

Comment:
If it's to be explicitly "remote to local", then fs.get() may be simpler, unless you really use features from rsync (e.g., testing the file sizes of files, deleting ones that disappeared on remote).